core/src/main/java/org/hibernate/ogm/util/impl/ResourceHelper.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.util.impl;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+public class ResourceHelper {
+
+	private static final String FILE_DELIMITER_PATTERN = "\\A";
```

Comment:
Is this a "file delimiter" or a "line delimiter"? Does it read multiple files?

Reply:
Are you going to change it? Otherwise I can do it before merging the PR. Let me know

Reply:
not yet

Reply:
Don't worry, I will do that

Reply:
thank you